ussologin/ussologin.go

```diff
@@ -32,9 +31,12 @@ var (
 	otpKey  = "Two-factor auth (Enter for none)"
 )
 
-// Login completes the login information using the provided filler
-// and attempts to obtain a USSO token using this information.
-func Login(filler form.Filler) (*usso.SSOData, error) {
+// GetToken uses filler to interact with the user and uses the provided
+// information to obtain an OAuth token from Ubuntu SSO. The returned
+// token can subsequently be used with LoginWithToken to perform a login.
+// The tokenName argument is used as the name of the generated token in
```

Comment:
I don't know what the rules are either. I don't think its worth digging through the ubuntu SSO code to find out. I expect we'll be fairly safe with anything we'll want to use.